xtext-latest.target

```diff
@@ -49,35 +49,143 @@
+					<type>jar</type>
+				</dependency>
+				<dependency>
+					<groupId>io.github.classgraph</groupId>
+					<artifactId>classgraph</artifactId>
+					<version>4.8.149</version>
+					<type>jar</type>
+				</dependency>
+				<dependency>
+					<groupId>org.apiguardian</groupId>
+					<artifactId>apiguardian-api</artifactId>
+					<version>1.1.2</version>
+					<type>jar</type>
+				</dependency>
+				<dependency>
+					<groupId>org.opentest4j</groupId>
+					<artifactId>opentest4j</artifactId>
+					<version>1.2.0</version>
+					<type>jar</type>
+				</dependency>
+				<dependency>
+					<groupId>org.ow2.asm</groupId>
+					<artifactId>asm</artifactId>
+					<version>9.5</version>
+					<type>jar</type>
+				</dependency>
+				<dependency>
+					<groupId>org.slf4j</groupId>
+					<artifactId>slf4j-simple</artifactId>
+					<version>1.7.31</version>
+					<type>jar</type>
+				</dependency>
+			</dependencies>
+		</location>
+		<location includeDependencyDepth="none" includeDependencyScopes="compile" includeSource="true" label="JUnit (Testing)" missingManifest="error" type="Maven">
+			<dependencies>
+				<dependency>
+					<groupId>org.junit.jupiter</groupId>
+					<artifactId>junit-jupiter-api</artifactId>
+					<version>5.9.1</version>
+					<type>jar</type>
+				</dependency>
+				<dependency>
+					<groupId>org.junit.jupiter</groupId>
+					<artifactId>junit-jupiter-engine</artifactId>
+					<version>5.9.1</version>
+					<type>jar</type>
+				</dependency>
+				<dependency>
+					<groupId>org.junit.jupiter</groupId>
+					<artifactId>junit-jupiter-migrationsupport</artifactId>
+					<version>5.9.1</version>
+					<type>jar</type>
+				</dependency>
+				<dependency>
+					<groupId>org.junit.jupiter</groupId>
+					<artifactId>junit-jupiter-params</artifactId>
+					<version>5.9.1</version>
+					<type>jar</type>
+				</dependency>
+				<dependency>
+					<groupId>org.junit.platform</groupId>
+					<artifactId>junit-platform-commons</artifactId>
+					<version>1.9.1</version>
+					<type>jar</type>
+				</dependency>
+				<dependency>
+					<groupId>org.junit.platform</groupId>
+					<artifactId>junit-platform-engine</artifactId>
+					<version>1.9.1</version>
+					<type>jar</type>
+				</dependency>
+				<dependency>
+					<groupId>org.junit.platform</groupId>
+					<artifactId>junit-platform-launcher</artifactId>
+					<version>1.9.1</version>
+					<type>jar</type>
+				</dependency>
+				<dependency>
+					<groupId>org.junit.platform</groupId>
+					<artifactId>junit-platform-runner</artifactId>
+					<version>1.9.1</version>
+					<type>jar</type>
+				</dependency>
+				<dependency>
+					<groupId>org.junit.platform</groupId>
+					<artifactId>junit-platform-suite-api</artifactId>
+					<version>1.9.1</version>
+					<type>jar</type>
+				</dependency>
+				<dependency>
+					<groupId>org.junit.platform</groupId>
+					<artifactId>junit-platform-suite-commons</artifactId>
+					<version>1.9.1</version>
```

Comment:
its not possbile to leverage bom here i assume

Reply:
Not yet, but there is a proposal to support properties in target-definitions, which could be used to only define the version once: https://github.com/eclipse-pde/eclipse.pde/issues/204

Reply:
Thinking about it a second time, Maven targets actually also support pom typed artifacts, that work as bom.
So it is probably already working. I'll check it and report back.

Reply:
OK, using boms is not yet possible, but it should not be too complicated to support it:
https://github.com/eclipse-m2e/m2e-core/issues/625

But it then will probably only be available in Tycho 4.